configure.py

```diff
@@ -449,8 +482,8 @@ def shell_escape(str):
             variables=[('varname', 'kBrowsePy')])
     n.newline()
 
-    objs += cxx('browse', order_only=built('browse_py.h'))
-    n.newline()
+    srcs += [cxxsrc('browse')]
+    order_only_deps['browse'] = built('browse_py.h')
```

Comment:
Another idea is to just drop this when doing amalgamation, since if the user can't be bothered to install ninja they certainly can't be bothered to install python.